### PR TITLE
added initial content for document-title-must-be-in-the-head-section

### PR DESCRIPTION
**Pull request by BarryBotha**

### CODE REVIEW

File: content/seo/document-title-must-be-in-the-head-section.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+Title tags `<title></title>` define the title of an HTML5 document. Title tags are important for user experience, social sharing and critical for SEO. They show up in the search engine SERPs (Search Engine Result Pages) and heavily influence whether users click through to your site. Title tags should be inside the document's `<head></head>` tags:
```
Comment (Johanndutoit) [inline content/seo/document-title-must-be-in-the-head-section.md]: Today I learned - SERP = Search Engine Result Pages

**Comment by Johanndutoit** [comment]
Users only receive this error if they have a title attribute but it's outside of head. Think you're trying to hit if no title tag is present ?

**Comment by BarryBotha** [comment]
Nope, I checked the code :)
I explain the tag and then mention it must be in the head.
In how to fix it is also clear.

**Comment by Johanndutoit** [comment]
Ok with this as is, still bring the same message across for me. If it causes confusion we can update later.

**Pull request merged**